File: Dante1/Module1.02/Zadanie2.01/main.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main()
{
    int num1, num2;
    printf("Enter numbers: ");

    int check = scanf("%i %i",&num1,&num2);
    if(check != 2)
    {
        printf("Incorrect input");
        return 1;
    }
    printf("%i",num1 + num2);
    return 0;
}

File: Dante1/Module1.02/Zadanie2.01/main.rs
fn main()
{
    println!("Enter two numbers: ");

    let mut input: String = String::new();
    std::io::stdin().read_line(&mut input).expect("Incorrect input");

    let mut input = input.trim().split_whitespace();

    let num1: i32 = input.next().expect("Too few arguments").parse().expect("Incorrect number");
    let num2: i32 = input.next().expect("Too few arguments").parse().expect("Incorrect number");
    
    println!("Result is: {}", num1 + num2);

    return;
}
